ui/pages/Project/components/FamilyTable/IndividualRow.jsx

```diff
@@ -127,8 +127,13 @@ const SHOW_DATA_MODAL_CONFIG = [
   },
 ]
 
+const BREAK_LINE_STYLING = { display: 'block' }
+
 const MmeStatusLabel = React.memo(({ title, dateField, color, individual, mmeSubmission }) => (
-  <Link to={`/project/${individual.projectGuid}/family_page/${individual.familyGuid}/matchmaker_exchange`}>
+  <Link
+    to={`/project/${individual.projectGuid}/family_page/${individual.familyGuid}/matchmaker_exchange`}
+    style={BREAK_LINE_STYLING}
```

Comment:
we do not use the style prop, we use styled components, unless there is an overwhelming compelling reason to do otherwise